ccr/createCCR.php

```diff
@@ -12,26 +12,32 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Common\Session\SessionUtil;
+use OpenEMR\Common\Twig\TwigContainer;
+use PHPMailer\PHPMailer\PHPMailer;
+
 // check if using the patient portal
 //(if so, then use the portal authorization)
 if (isset($_GET['portal_auth'])) {
     $landingpage = "../portal/index.php";
 
     // Will start the (patient) portal OpenEMR session/cookie.
     require_once(dirname(__FILE__) . "/../src/Common/Session/SessionUtil.php");
-    OpenEMR\Common\Session\SessionUtil::portalSessionStart();
+    SessionUtil::portalSessionStart();
 
     if (isset($_SESSION['pid']) && isset($_SESSION['patient_portal_onsite_two'])) {
         $pid = $_SESSION['pid'];
         $ignoreAuth = true;
         global $ignoreAuth;
     } else {
-        OpenEMR\Common\Session\SessionUtil::portalSessionCookieDestroy();
+        SessionUtil::portalSessionCookieDestroy();
         header('Location: ' . $landingpage . '?w');
         exit;
     }
 } else {
     // Check authorization.
```

Comment:
this is a weird block (try best to never include globals.php twice). Would just have this block set up a flag such as notPatientPortal = true

Then use that flag to run the auth after the includes below (so will then be below the globals.php below)